tests/net/vlan/testcase.yaml

```diff
@@ -15,3 +15,6 @@ tests:
     extra_configs:
       - CONFIG_NET_L2_ETHERNET_RESERVE_HEADER=y
       - CONFIG_NET_BUF_VARIABLE_DATA_SIZE=y
+      - platform:mimxrt1060_evk@C/mimxrt1062/qspi:CONFIG_TEST_EXTRA_STACK_SIZE=1024
```

Comment:
I'd consider increasing the ZTEST stack size for this test suite globally. Those tests are generic so if there's a stack overflow on one platform, others with the same architecture (and possibly even more) should have the same issue. Addressing this platform-by-platform just doesn't scale.

Reply:
ok, let me add.